tools/__tasks__/compile/javascript/rjs.config.js

```diff
@@ -7,31 +7,31 @@
         facia: 'projects/facia',
         membership: 'projects/membership',
         commercial: 'projects/commercial',
-        bean: 'components/bean/bean',
-        bonzo: 'components/bonzo/bonzo',
-        domReady: 'components/domready/ready',
-        EventEmitter: 'components/eventEmitter/EventEmitter',
-        fastdom: 'components/fastdom/index',
-        fence: 'components/fence/fence',
-        lodash: 'components/lodash-amd',
+        bean: '../../../static/vendor/javascripts/components/bean/bean',
```

Comment:
Since `baseUrl` is:
```
../../../../static/src/javascripts
```
is this path equivalent to:
```
../../vendor/javascripts/components/bean/bean
```
?